assets/capability/capabilities/speaker_playing.json

```diff
@@ -58,7 +64,9 @@
           "sv": "Spelas !{{|inte}}",
           "no": "Avspilles !{{|ikke}}",
           "es": "!{{Se está|No se está}} reproduciendo",
-          "da": "Spiller !{{|ikke}}"
+          "da": "Spiller !{{|ikke}}",
+          "ru": "!{{|not}} воспроизводится",
```

Comment:
Seems like the Russian translator did not know `{{ <string> }}` needs to be translated

Reply:
```suggestion
          "ru": "!{{|не}} воспроизводится",
```